library/ssl_tls13_client.c

```diff
@@ -35,7 +35,7 @@
 #include "ssl_debug_helpers.h"
 #include "md_psa.h"
 
-#if defined(PSA_WANT_ALG_ECDH) || defined(PSA_WANT_ALG_FFDH)
+#if defined(MBEDTLS_KEY_EXCHANGE_SOME_XXDH_ANY_PSA_ENABLED)
```

Comment:
This is a 1.3 only file, I'd prefer using a 1.3 macro, namely `MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_SOME_EPHEMERAL_ENABLED` here.

Edit: multiple occurrences in this file, `tls13_server` and `tls13_client`.